lib/src/components/Store.test.ts

```diff
@@ -28,11 +29,34 @@ describe('Store', () => {
     expect(uut.getComponentClassForName('example.mycomponent')).toEqual(MyWrappedComponent);
   });
 
-  it('clean by component id', () => {
+  it('clean props by component id', () => {
     uut.setPropsForId('refUniqueId', { foo: 'bar' });
+    uut.cleanId('refUniqueId');
```

Comment:
Maybe rename `cleanId` to `clearProps` ?

Reply:
I'm changing it to `clearComponent` because it clears the props and the component from the store.